apps/addons/dashy.yml

```diff
@@ -0,0 +1,34 @@
+---
+version: '3'
+services:
+  dashy:
+    hostname: 'dashy'
+    container_name: 'dashy'
+    environment:
+      - 'PGID=${ID}'
+      - 'PUID=${ID}'
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+    image: 'lissy93/dashy:latest'
+    restart: '${RESTARTAPP}'
+    networks:
+      - proxy
+    security_opt:
+      - 'no-new-privileges:true'
+    volumes:
+      - '${APPFOLDER}/dashy/conf.yml:/app/public/conf.yml:rw'

```

Comment:
@DrAg0n141 Please fix this!

Reply:
Thiw will be super hard to code 
It's so massively customizable parts 

So I thinks is mus show the link  and  the after press enter just create a empty file and open them with  

```sh
sudo nano ${APPFOLDER}/dashy/conf.yml
```

All other should be made to much problems and you lose the custome parts